lib/lhm/chunk_insert.rb

```diff
@@ -11,7 +12,7 @@ def initialize(migration, connection, lowest, highest, options = {})
     end
 
     def insert_and_return_count_of_rows_created
-      @connection.update(sql, @retry_options)
+      @connection.update(Lhm::ProxySQLHelper.tagged(sql), @retry_options)
```

Comment:
Tagging the `insert ignore`. Might be seen as unnecessary, since if you're trying to insert on a replica it would throw an error regardless, but it would also make sense to tag "critical" queries. Any thoughts?